src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,45 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int STORAGE_MAX_SIZE = 10;
+    private K[] keys;
+    private V[] values;
+    private int counterOfSpace;
+
+    public StorageImpl(Object pair) {
+        keys = (K[])new Object[STORAGE_MAX_SIZE];
+        values = (V[])new Object[STORAGE_MAX_SIZE];
+    }
```

Comment:
You have changed tests for this parameter but not even using it anywhere
```suggestion
    public StorageImpl() {
        keys = (K[]) new Object[STORAGE_MAX_SIZE];
        values = (V[]) new Object[STORAGE_MAX_SIZE];
    }
```